src/main/kotlin/uk/gov/justice/digital/hmpps/visitscheduler/dto/MigrateVisitRequestDto.kt

```diff
@@ -26,6 +27,9 @@ data class MigrateVisitRequestDto(
   @Schema(description = "Visit Status", example = "RESERVED", required = true)
   @field:NotNull
   val visitStatus: VisitStatus,
+  @Schema(description = "Outcome Status", example = "RESERVED", required = true)
+  @field:NotNull
+  val outcomeStatus: OutcomeStatus ? = OutcomeStatus.NOT_RECORDED,
```

Comment:
dont think I need this either leave default on Entity only?

Reply:
What's the need to default? seems like a change to the feature